data/migrations/20190311030207_notifications.js

```diff
@@ -1,15 +1,16 @@
 exports.up = function(knex, Promise) {
-  return knex.schema.createTable("watering", table => {
+  return knex.schema.createTable("notifications", table => {
     table.increments();
-    table.datetime("wateringTime");
+    table.datetime("notificationsTime");
     table.boolean("smsDelivered").defaultTo(false);
-    table.integer("plantId")
+    table
+      .integer("plantId")
       .references("id")
       .inTable("plants")
       .onDelete("CASCADE");
   });
 };
```

Comment:
Let add a `userId` foreign key requirement to the `notifications` table. It will make it easier to query all notifications by userId